event/event.gen.go

```diff
@@ -53,14 +61,6 @@ func AudioEnd(listener func(*vecty.Event)) *vecty.EventListener {
 	return &vecty.EventListener{Name: "audioend", Listener: listener}
 }
 
-// AudioProcess is an event fired when the input buffer of a
-// ScriptProcessorNode is ready to be processed.
-//
-// https://developer.mozilla.org/docs/Web/Events/audioprocess
-func AudioProcess(listener func(*vecty.Event)) *vecty.EventListener {
```

Comment:
It seems like this has gotten removed by the generator even though it is still a valid event (not deprecated etc). We'll need to figure out / fix why this got removed before we can merge this.

Reply:
@slimsag apologies for the late feedback, I was  in a hospital , I feel better now. I t seems the cause is the `.icon-thumbs-down-alt` class see https://github.com/gopherjs/vecty/pull/195/files#diff-7f46ba42e4ad2574c9cbe6836be77d61L140  I don't know why the logic is there. Maybe the docs added that icon for some reason.

Reply:
Sorry to hear you were not feeling well! Hope you're feeling better.

I just had a chance to look at this, and it looks like the problem is just more inconsistency on the MDN -_-

![image](https://user-images.githubusercontent.com/3173176/39090672-4e168632-459a-11e8-888c-ae168463ef98.png)

The `audioprocess` event is not marked as deprecated, but the data type it events `AudioProcessingEvent` is deprecated. Obviously that is silly because that means both are deprecated, so we're doing the right thing here by removing this (since we don't keep deprecated items in here). Just another reason to solve #136 soon.